.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+php:
+  - 7.1
+
+cache:
+  directories:
+    - ~/.composer/cache
+    - $HOME/.npm
+
+env:
+  - ELASTIC_SEARCH_INDEX_PREFIX=""
+
+before_install:
+  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - phpenv config-rm xdebug.ini || return 0
+
+install:
+  - composer global require hirak/prestissimo
+  - COMPOSER_MEMORY_LIMIT=-1 travis_retry composer install --optimize-autoloader --no-interaction --no-progress --dev
```

Comment:
`COMPOSER_MEMORY_LIMIT` is an environment variable too, why not define it next to the `ELASTIC_SEARCH_INDEX_PREFIX`

Reply:
Ok i moved that. It was because i found that command on internet :D

Reply:
Oh after i moved `COMPOSER_MEMORY_LIMIT` to env parameters travis started to fail on composer, so i moved that back

Reply:
OK